docs/book/src/migration/manually_migration_guide_gov3_to_gov4.md

```diff
@@ -35,34 +41,164 @@ layout:
-- Replace the import `admissionv1beta1 "k8s.io/api/admission/v1beta1"` with `admissionv1 "k8s.io/api/admission/v1"` in the webhook test files
+##### Now, let's perform the layout changes to comply within:
+
+- The directory `api` was renamed to `apis` to follow the standard applied within controllers
+- The `api(s)` directory has been moved under a new directory called `pkg`
+- The `controller(s)` directory has been moved under a new directory called `internal`
+- The `main.go` previously scaffolded in the root directory has been moved under a new directory  called `cmd`
+
+Therefore, you can check the changes in the layout results into:
+
+```sh
+...
+├── cmd
+│ └── main.go
+├── internal
+│ └── controllers
+└── pkg
+    └── apis
+```
+
+##### In this way, you will need to:
+
+- Create a new directory `cmd` and move the `main.go` under it.
+- If your project does not support multi-group the APIs are scaffold under a directory called `api`. Rename this diretory to `apis`
+- Move the `apis` directory under the `pkg`
+- Move the `controllers` directory under the `internal` 
+- Now ensure that the imports will be updated accordingly by:
+  - Update the `main.go` imports to look for the new path of your controllers under the `pkg` directory
+  - Update all controllers imports to look for the new path of your apis under the `pkg` directory
+
+**Then, let's update the scaffolds paths**
+
+- Update the Dockerfile by replacing:
+
+```
+# Copy the go source
+COPY main.go main.go
+COPY apis/ apis/
+COPY controllers/ controllers/
+```
+
+With: 
+
+```
+COPY cmd/main.go cmd/main.go
+COPY pkg/apis/ pkg/apis/
+COPY internal/controllers/ internal/controllers/
+```
+
+Then, replace:
+
+```
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o manager main.go
+
+```
+
+With:
+
+```
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o manager cmd/main.go
+```
+
+- Update the Makefile targets to build and run the manager by replacing:
+
+```
+.PHONY: build
+build: manifests generate fmt vet ## Build manager binary.
+	go build -o bin/manager main.go
+
+.PHONY: run
+run: manifests generate fmt vet ## Run a controller from your host.
+	go run ./main.go
+```
+
+With:
+
+```
+.PHONY: build
+build: manifests generate fmt vet ## Build manager binary.
+	go build -o bin/manager cmd/main.go
+
+.PHONY: run
+run: manifests generate fmt vet ## Run a controller from your host.
+	go run ./cmd/main.go
+```
+
+- Update the `internal/controllers/suite_test.go` to set the path for the `CRDDirectoryPaths`:
+
+Replace:
+
+```
+CRDDirectoryPaths:     []string{filepath.Join("..", "config", "crd", "bases")},
+```
+
+With:
+
+```
+CRDDirectoryPaths:     []string{filepath.Join("..", "..", "config", "crd", "bases")},
+```
+
+Note that if your project has multiple groups (`multigroup:true`) then the above updated should result into `"..", "..", "..",` instead of `"..",".."`
```

Comment:
```suggestion
Note that if your project has multiple groups (`multigroup:true`) then the above update should result into `"..", "..", "..",` instead of `"..",".."`
```